# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserParent.h
# Commit: f45996fe15b2
# Full Hash: f45996fe15b23696c88f85a6b154146d3717fc58
# Author: Andrew McCreight <continuation@gmail.com>
# Date: 2021-05-06 03:49:11
# Regressor Bug: 1618547
# File Overlap Count: 1
# Description:
#   Bug 1618547 - Support Fission in the process priority manager. r=kmag,gsvelto
#   
#   With Fission, there can be multiple BrowserParents in a single tab, so
#   this patch moves the tracking of active tabs onto the top BrowsingContext
#   in a tab. If the priority of a top BC is changed, then the activity
# ==============================================================================

diff -r 0e0f90271853 -r f45996fe15b2 dom/ipc/BrowserParent.h
--- a/dom/ipc/BrowserParent.h	Wed May 05 20:07:07 2021 +0000
+++ b/dom/ipc/BrowserParent.h	Wed May 05 20:29:14 2021 +0000
@@ -702,8 +702,6 @@
   void PreserveLayers(bool aPreserveLayers);
   void NotifyResolutionChanged();
 
-  void Deprioritize();
-
   bool StartApzAutoscroll(float aAnchorX, float aAnchorY, nsViewID aScrollId,
                           uint32_t aPresShellId);
   void StopApzAutoscroll(nsViewID aScrollId, uint32_t aPresShellId);
@@ -969,9 +967,6 @@
   // and have uploaded - for that, use mHasLayers.
   bool mRenderLayers : 1;
 
-  // Whether this is active for the ProcessPriorityManager or not.
-  bool mActiveInPriorityManager : 1;
-
   // True if the compositor has reported that the BrowserChild has uploaded
   // layers.
   bool mHasLayers : 1;